CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,134 @@
+# Contributor Covenant Code of Conduct
+
+## Our Pledge
+
+We as members, contributors, and leaders pledge to make participation in our
+community a harassment-free experience for everyone, regardless of age, body
+size, visible or invisible disability, ethnicity, sex characteristics, gender
+identity and expression, level of experience, education, socio-economic status,
+nationality, personal appearance, race, caste, color, religion, or sexual
+identity and orientation.
+
+We pledge to act and interact in ways that contribute to an open, welcoming,
+diverse, inclusive, and healthy community.
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment for our
+community include:
+
+- Demonstrating empathy and kindness toward other people
+- Being respectful of differing opinions, viewpoints, and experiences
+- Giving and gracefully accepting constructive feedback
+- Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+- Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+Examples of unacceptable behavior include:
+
+- The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+- Trolling, insulting or derogatory comments, and personal or political attacks
+- Public or private harassment
+- Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+- Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Enforcement Responsibilities
+
+Community leaders are responsible for clarifying and enforcing our standards of
+acceptable behavior and will take appropriate and fair corrective action in
+response to any behavior that they deem inappropriate, threatening, offensive,
+or harmful.
+
+Community leaders have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct, and will communicate reasons for moderation
+decisions when appropriate.
+
+## Scope
+
+This Code of Conduct applies within all community spaces, and also applies when
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to Daniel Roe directly at daniel@roe.dev. If you want to report
+unacceptable behavior by Daniel Roe, please contact Sébastien Chopin
+instead at seb@nuxtjs.com.
```

Comment:
Why/How to individuals are selected instead of a team account (same as security)

Reply:
To make the process more transparent. If someone wants to report an issue _with_ one of the project team they need to know whom they are making the report to.

Reply:
Shall be seb@atinux.com

Reply:
Sure but I still disagree with including individual names in Code-Of-Conduct. Individual contact shouldn't be the first option. This way it takes away internal transparency within the nuxt team :)

In case of unacceptable behavior from the lead, we could refer to the governance guide (which is still not updated, to remind you!) for authors and individual alternative contact points however I'm pretty sure it should be an unlikely situation.

Reply:
I disagree. This isn't about increasing transparency within the nuxt team (as you say, it's surely an unlikely scenario) but providing the safest possible way for users to report issues even in unlikely scenarios. It is regarded as best practice to have named contacts.

Reply:
Then please include alex and my email as well ;)

Reply:
As you wish, I've added you and Alex as well, so we just have a list of four people.

Reply:
Thanks!